kalliope/core/SynapseLauncher.py

```diff
@@ -48,6 +55,45 @@ def start_synapse_by_name(cls, name, brain=None, overriding_parameter_dict=None)
             lifo_buffer.add_synapse_list_to_lifo(list_synapse_to_process)
             return lifo_buffer.execute(is_api_call=True)
 
+    @classmethod
+    def start_synapse_by_list_name(cls, list_name, brain=None, overriding_parameter_dict=None, new_lifo=False):
```

Comment:
Why not reusing the method "start_synapse_by_name" for each entry of the list_name ?

Reply:
because if you do that, a new lifo will be created for each new synapse to execute. 
here, each synapse of the provided list is added to the same lifo.